src/minidump.rs

```diff
@@ -304,7 +304,7 @@ fn read_string_utf16(
 }
 
 /// Convert `bytes` with trailing NUL characters to a string
-fn string_from_bytes_nul(bytes: &[u8]) -> Option<Cow<str>> {
+fn string_from_bytes_nul(bytes: &[u8]) -> Option<Cow<'_, str>> {
```

Comment:
Is this really a thing that `cargo fix` does? I don't see why lifetime elision isn't a perfectly sensible thing here.

Reply:
Correct. There's some information on the reasoning behind that on [the anonymous lifetime](https://doc.rust-lang.org/edition-guide/rust-2018/ownership-and-lifetimes/the-anonymous-lifetime.html#more-details) docs. Essentially, for references it is clear that there is an elided lifetime, but it is not for types. Therefore, one should annotate the anonymous lifetime always, to show that the type is not `'static`.

Reply:
Okay, that's very reasonable. Thanks for digging up the link!